README.md

```diff
@@ -33,6 +33,88 @@ updating the typesystems as well.
 
 # Building
 
+## General
+
+Building PythonQt requires a couple of steps.
+Follow these instructions in order to get a correctly built PythonQt runtime and Qt bindings.
+
+### ⚠️ Pregenerated Bindings Warning ⚠️
+
+It is highly recommended to build the Qt bindings yourself and not(!) use the pregenerated ones. To ensure that, delete the pregenerated directory starting with `generated_cpp_*`.
+Do not build `PythonQt.pro` directly because it will only use the pregenerated bindings!
```

Comment:
Actually, it first checks for the path "generated_cpp", assuming that your self-generated sources are there, and only if it does not exist it checks for the other paths (see `common.prf`).

Reply:
Indeed. No need to remove any pre-generated bindings. The generated_cpp will be used if created by user. I once tried to generate bindings and failed, so I've used 5.6 pre-gen bindings for a long time instead.

Reply:
I still think it is a valid point - it is always preferrable to use the bindings generated for your concrete version instead of a pre-generated one for an older version. That generating the bindings did not work reliably before was one reason to use the pre-generated bindings that is now obsolete.
So maybe lessen the tone of the recommendation a bit ("It is recommended to build the Qt bindings yourself and not use the pregenerated ones."), and add a bit of a rational.

Reply:
I've lessen the tone.

Reply:
> Do not build `PythonQt.pro` directly because it will only use the pregenerated bindings!

This is partly true, in the sense that while it does build the generator first, it does not generate the bindings, so if you haven't built them before it will indeed use the checked-in ones. 
I think you can leave it as is, but ultimately PythonQt.pro shall be adapted to also generate the bindings, maybe depending on some variable (or add the CMakeLists.txt files, or both).